fluss-lake/fluss-lake-paimon/src/test/java/com/alibaba/fluss/lake/paimon/flink/FlinkUnionReadLogTableITCase.java

```diff
@@ -97,6 +99,43 @@ void testReadLogTableFullType(boolean isPartitioned) throws Exception {
         assertThat(actual).containsExactlyInAnyOrderElementsOf(expected);
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {false, true})
+    void testReadLogTableInStreamingMode(boolean isPartitioned) throws Exception {
+        // first of all, start tiering
+        JobClient jobClient = buildTieringJob(execEnv);
+
+        String tableName = "logTable_" + (isPartitioned ? "partitioned" : "non_partitioned");
```

Comment:
the table has same name with tables in method `testReadLogTableFullType`, I'm afraid of there will be conflicts. I suggest you to use a new name.